prover/witness_generator/Cargo.toml

```diff
@@ -12,6 +12,7 @@ publish = false # We don't want to publish our binaries.
 
 [dependencies]
 zksync_dal = { path = "../../core/lib/dal" }
+zksync_core = { path = "../../core/lib/zksync_core" }
```

Comment:
maybe just copy-paste storage oracle impl to prover/witness-generator? We will remove witness generator from zksync-core, so it has to be done anyway

Reply:
okay